.teamcity/_self/projects/WebApp.kt

```diff
@@ -733,7 +733,7 @@ object RunCalypsoPlaywrightE2eTests : BuildType({
 				export PLAYWRIGHT_BROWSERS_PATH=0
 
 				# Install modules
-				${_self.yarn_install_cmd}
+				yarn install
```

Comment:
Could we add a comment about why we need to go with the raw yarn install command?

Reply:
Should we be skipping dev & optional deps here? Also, can we use the pure/frozen lockfile option here? i.e.

```suggestion
				yarn install --frozen-lockfile --production --ignore-optional
```

cc @scinos

Reply:
I've incorporated the suggested changes here. Please take a look!

Reply:
I think we can just keep it with `yarn install`, that's what we use everywhere else: https://github.com/Automattic/wp-calypso/blob/7efc3246ac99b30f24a6040fc4bfeaf3791f7d10/.teamcity/_self/yarnInstall.kt#L16

Also, I think we need to install dev deps (and possibly optional ones too). Wouldn't something like test utils be added to dev dependencies?

Reply:
Actually, @worldomonation, I think a better way to solve this problem would be to set the `use_cached_node_modules` parameter to false for this build type.

Reply:
You can just add this after the "description" property in the build type:
```kt
params {
	param("use_cached_node_modules", "false")
}
```

Reply:
@noahtallen Like this I suppose?

```
	description = "Runs Calypso e2e tests using Playwright"
	param("use_cached_node_modules", "false")
```

Reply:
yep, exactly!